lib/code_repository.ts

```diff
@@ -0,0 +1,16 @@
+import { Client, Exists, Match, Index } from "faunadb";
+
+const client = new Client({
+  secret: process.env.FAUNADB_SECRET,
+  domain: "db.us.fauna.com",
+});
+
+const useCodeRepository = () => {
+  const isValid = async (code: string) => {
```

Comment:
I think this is going to have to take in the repository name or something. There will be a different list for each POD nft:
- red pil
- green pil
- spork and sour

Reply:
done